contracts/Strategy.sol

```diff
@@ -186,21 +175,29 @@ contract Strategy is BaseStrategyInitializable {
         _startCooldown();
     }
 
+    function harvestTrigger(uint256 callCostInWei) public view override returns (bool) {
+        CooldownStatus cooldownStatus = _checkCooldown();
+        return (cooldownStatus == CooldownStatus.Claim) && super.harvestTrigger(callCostInWei);
+    }
+
     function liquidatePosition(uint256 _amountNeeded)
         internal
         override
         returns (uint256 _liquidatedAmount, uint256 _loss)
     {
-        uint256 freeAssets = balanceOfWant();
-        _liquidatedAmount = freeAssets;
+        _liquidatedAmount = balanceOfWant();
+
+        if (_amountNeeded > _liquidatedAmount) {
+            _loss = _amountNeeded.sub(_liquidatedAmount);
```

Comment:
Is there a reason to take a loss here? The funds shouldt be held as stkAave temporarily.